about.html

```diff
@@ -0,0 +1,109 @@
+                children’s literature is a recurring theme throughout this work. Young children don’t necessarily see
+                animals as different. They see them as an extension of their human world and don’t recognize a
+                difference. The artists attempt to regain this sense of “extension” in their work. Animals don’t feature as
+                secondary characters but play an equal role in the visual interactions of these artworks.</p>
+            <p class="contact-us">Please contact us via email if you have any questions about the art exhibition at <a href="#">moodeonline@gmail.com</a></p>
+        </div>
+    </section>
+    <!-- Main program section -->
+    <section class="about-logo">
+        <h2 class="logo-info">Moode Art Enterprise Logo</h2>
+        <p class="logo-descript">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sed a quo eius reiciendis veniam mollitia tempore quasi ea veritatis perferendis.</p>
+        <div class="imglogo">
+            <img class="logo-img" src="./Images/Moode-logo.JPG" alt="logo description">
+        </div>
+    </section>
+    <section class="past-exhibitions">
+        <h2 class="past-exhibitions-title">See Previous Exhibitions</h2>
+        <span class="small-line"></span>
+        <p class="pe-intro">Kindly find below our recent exhibitions in Accra and London</p>
+        <div class="exhibit-cont">
+            <div class="first-exhibit">
+                <div id="overlay1"></div>
+                <a id="imglink" href="#">2020</a>
+                <a id="imglink2" href="#">An art exhibition in Accra</a>
+            </div>
+            <div class="second-exhibit">
+                <div id="overlay2"></div>
+                <a id="imglink" href="#">2021</a>
+                <a id="imglink2" href="#">An art exhibition in London</a>
+            </div>
+        </div>
+    </section>
+ 
+    <footer class="footer-section-about">
+        <div class="partner-section">
+            <h2 class="program">Partners</h2>
+            <span class="small-line"></span>
+            <div class="partner-list">
+                <ul class="partners">
+                    <li><a href="#"><img class="partner-img" src="./Images/masterworks.jpeg"></a></li>
+                    <li><a href="#"><img class="partner-img" src="./Images/UiPath_Logo.png"></a></li>
+                    <li><a href="#"><img class="partner-img" src="./Images/heb-image.png"></a></li>
+                    <li><a href="#"><img class="partner-img" src="./Images/Logo_tado_GmbH.jpeg"></a></li>
+                    <li><a href="#"><img class="partner-img" src="./Images/Wahed_Investment_Firm.jpeg"></a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="sign-off">
+            <p class="comp-name">@ Moode Online Enterprise</p>
+            <p class="comp-sig"> 2022 Moode Art Global. All rights reserved.</p>
+        </div>
```

Comment:
Great job @Jaymelfah ✋

- [ ] Kindly remove the partner section from your about page. This section does not exist in the design provided.
- [ ] You may also change the background of the footer on the about section to a dark color just like how it appears in the design given.
- Your design
![partners](https://user-images.githubusercontent.com/63917725/190085075-a1514a78-9f2a-4d43-ae0a-545f4c08e8f7.PNG)

- Provided design
![provided](https://user-images.githubusercontent.com/63917725/190085264-d36306fe-2bd5-474b-b06e-52223f0754b2.PNG)